src/Maestro/tests/Maestro.ScenarioTests/TestHelpers.cs

```diff
@@ -152,6 +152,14 @@ internal static string FormatExecutableCall(string executable, params string[] a
 
             return output.ToString();
         }
+
+        public static string ParseOutSpecificRepoPolicy(string allReturnedPolicies, string repoUrlAndBranch)
+        {
+            int startingIndex = allReturnedPolicies.IndexOf(repoUrlAndBranch);
+            string removeEarlierPolicies = allReturnedPolicies.Substring(startingIndex);
+            string[] removeLaterPolicies = removeEarlierPolicies.Split(']');
+            return removeLaterPolicies[0] + ']';
```

Comment:
I think you're cutting off the final \r\n that the tests are expecting in LN53 of the RepoPolicies test.

Reply:
Thanks for catching that! I realized I could change the channel name back to actually run the full test to test this (oops), so I'm doing a little clean up there.